Messaging.md

```diff
@@ -379,15 +395,21 @@ See [the Oracle specifications](./Oracle.md#oracle-event) for more details.
 This type contains an `oracle_event` and a signature certifying its origination.
 See [the Oracle specifications](./Oracle.md#oracle-announcements) for more details.
 
-#### Version 0 `oracle_announcement`
+#### Version 1 `oracle_announcement`
 
-1. type: 55332 (`oracle_announcement`)
+1. type: 55354 (`oracle_announcement`)
 2. data:
-   * [`signature`:`annoucement_signature`]
-   * [`x_point`:`oracle_public_key`]
+   * [`signature`:`annoucement_pok_signature`]
+   * [`signature`:`attestation_pok_signature`]
+   * [`u16`:`nb_nonces`]
+   * [`nb_nonces*signature`:`nonce_pok_signatures`]
+   * [`x_point`:`oracle_announcement_public_key`]
+   * [`x_point`:`oracle_attestation_public_key`]
+   * [`nb_nonces*x_point`:`oracle_nonces`]
```

Comment:
nit: might be good to have that before the `nonce_pok_signatures` in case someone would want to do de-serialization and validation at the same time (I don't think I would but it also seems like a tiny change)